### PR TITLE
Add RetroArch (https://github.com/webosbrew/RetroArch)

### PR DESCRIPTION
**Pull request by Ruthenic**
I believe that the package specification is okay? I'm not entirely sure if you can test them or something, but it appears to be fine

### CODE REVIEW

File: packages/com.retroarch.yml [hunk 1]
```diff
@@ -0,0 +1,283 @@
+title: Retroarch
+iconUri: https://github.com/webosbrew/RetroArch/blob/webos-tv-support/webos/icon160.png
```
Comment (mariotaku) [inline packages/com.retroarch.yml:2]: Could you change to `https://github.com/webosbrew/RetroArch/raw/webos-tv-support/webos/icon160.png`? I think this link will be resolved to image file.

File: packages/com.retroarch.yml [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+manifestUrl: https://github.com/webosbrew/RetroArch/releases/download/v1.9.1-webos.0/com.retroarch.manifest.json
```
Comment (mariotaku) [inline packages/com.retroarch.yml:3]: Could you change to `https://github.com/webosbrew/RetroArch/releases/latest/download/com.retroarch.manifest.json`?

**Review by mariotaku — CHANGES_REQUESTED** [review]
A little bit changes and we are good to go.

**Comment by mariotaku** [comment]
Maybe I can also implement CI check for new entries.

File: packages/com.retroarch.yml (hunk 2 above)
Comment (Ruthenic) [reply packages/com.retroarch.yml]: Changed.

File: packages/com.retroarch.yml (hunk 1 above)
Comment (Ruthenic) [reply packages/com.retroarch.yml]: changed

**Comment by Ruthenic** [comment]
everything *should* be fixed

**Review by mariotaku — APPROVED** [review]

**Pull request merged**